server/app/views/support/UnsupportedBrowserView.java

```diff
@@ -0,0 +1,58 @@
+package views.support;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.h1;
+import static j2html.TagCreator.img;
+
+import j2html.tags.ContainerTag;
+import javax.inject.Inject;
+import play.twirl.api.Content;
+import views.BaseHtmlView;
+import views.HtmlBundle;
+import views.applicant.ApplicantLayout;
+import views.style.StyleUtils;
+import views.style.Styles;
+
+public class UnsupportedBrowserView extends BaseHtmlView {
```

Comment:
Not blocking: Consider preemptively marking this as final to make the choice to allow subclassing explicit rather than implicit.

Reply:
Done.